### PR TITLE
Fixed the unexpected behaviour due to `--assembly` parameter of `pairtools parse`

### PR DESCRIPTION
**Pull request by SaimMomin12**

### CODE REVIEW

File: tools/pairtools/parse.xml [hunk 1]
```diff
@@ -8,7 +8,9 @@
         pairtools parse
             '$sam_path'
             -c '$chroms_path'
-            $assembly
+            #if str( $assembly_selection.assembly_name_selector ) == "yes":
+                --assembly $assembly_name
```
Comment (bgruening) [inline tools/pairtools/parse.xml:12]: assembly_name needs to be single-quoted as it is a text param

File: tools/pairtools/parse.xml [hunk 2]
```diff
@@ -27,7 +29,16 @@
     <inputs>
         <param name="sam_path" type="data" format="sam,bam" label="Input SAM/BAM file" help="Input SAM or BAM file with paired-end sequence alignments of Hi-C molecules."/>    
         <param name="chroms_path" argument="-c" type="data" format="tabular" label="Input a chromosome order file" help="Chromosome order used to flip interchromosomal mates. Any scaffolds not listed will be ordered lexicographically."/>
-        <param name="assembly" type="text" value="" label="Input a name of a genome assembly" help="Name of genome assembly (e.g. hg19, mm10) to store in the pairs header"></param>
+        <conditional name="assembly_selection">
```
[the remark this answers is not in the document]
Comment (SaimMomin12) [reply tools/pairtools/parse.xml:32]: Earlier, when a assembly name was defined the tool threw error due to missing parameter `--assembly`, thats why I thought of making it conditional here. 

Perhaps Is there a way, when the user defined a custom assembly name - it should add --assembly $assembly_name to the command?

[the remark this answers is not in the document]
Comment (SaimMomin12) [reply tools/pairtools/parse.xml:32]: Okay I figured it out - It was rather so straightforward :smile:

File: tools/pairtools/parse.xml [hunk 3]
```diff
@@ -8,7 +8,9 @@
         pairtools parse
             '$sam_path'
             -c '$chroms_path'
-            $assembly
+            #if $assembly_name:
```
Comment (bgruening) [inline tools/pairtools/parse.xml:11]:
```suggestion
            #if str($assembly_name).strip(): 
```

**Review by bgruening — COMMENTED** [review]
last but not least a version bump of the suffix is needed or manually at the version of the particular parse tool

**Comment by SaimMomin12** [comment]
Thanks @bgruening  💯

**Pull request merged**